### DISCUSSION TITLE
为什么1.16.5离线游戏无法进入多人服务器

### DISCUSSION THREAD

**Discussion by steamboat635441** [Meloong-Git/PCL#2003] [提问] [answered]
离线游戏无法进入多人，显示需求Microsoft账号

**Comment by WForst-Breeze** [comment]
离线模式无法进入开启正版验证的服务器诸如 Hypixel 等，如果您是与朋友联机，可以请求房主安装 [Lan Server Properties](https://www.curseforge.com/minecraft/mc-mods/lan-server-properties) 模组以允许离线玩家进入游戏

**Reply by steamboat635441** [reply]
并非是服务器开了正版验证，主菜单的多人游戏选项被直接禁止无法进入

**Reply by WForst-Breeze** [reply, marked as answer]
经测试与查阅资料，近期 1.16.4 及 1.16.5 均无法使用离线模式进入多人游戏。
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/f9620d50-79ae-4eec-b571-8e3f74be0648)

最简单粗暴的解决方案为：断网打开 PCL2 并启动 1.16.5，问题即可解决。然后记得再连上网，这样即可正常多人游戏。

###### 还有一种相对稳定的解决方法，可以看看[这篇博客](http://ylcao.top/2023/06/09/Minecraft%E7%A6%81%E6%AD%A2%E5%A4%9A%E4%BA%BA%E6%B8%B8%E6%88%8F%E7%A0%B4%E8%A7%A3/)。同时，这篇博客也进行了相关源代码分析

**Reply by WorldHim** [reply]
> 经测试与查阅资料，近期 1.16.4 及 1.16.5 均无法使用离线模式进入多人游戏。 ![image](https://user-images.githubusercontent.com/110760354/244611943-f9620d50-79ae-4eec-b571-8e3f74be0648.png)
> 
> 最简单粗暴的解决方案为：断网打开 PCL2 并启动 1.16.5，问题即可解决。然后记得再连上网，这样即可正常多人游戏。
> 
> ###### 还有一种相对稳定的解决方法，可以看看[这篇博客](http://ylcao.top/2023/06/09/Minecraft%E7%A6%81%E6%AD%A2%E5%A4%9A%E4%BA%BA%E6%B8%B8%E6%88%8F%E7%A0%B4%E8%A7%A3/)。同时，这篇博客也进行了相关源代码分析

目前已有模组解决此问题，见：
Forge & Fabric：[MCTeamPotato/MultiOfflineFix](https://github.com/MCTeamPotato/MultiOfflineFix)
Fabric：[ChickenPige0n/offline-multiplayer-fabric](https://github.com/ChickenPige0n/offline-multiplayer-fabric)
